Player.gd

```diff
@@ -48,21 +48,55 @@ func get_player_input():
 			$AnimationPlayer.play("Attack")
 			$Networking.sync_character_animation = "Attack"
 			
-			rpc_id(1, "spawn_projectile")
+			rpc_id(1, "spawn_projectile_server")
```

Comment:
The client that wants to shoot calls the _server method, to tell the server they're shooting.